docs/platforms/javascript/common/enriching-events/scopes/index.mdx

```diff
@@ -102,7 +102,7 @@ Sentry.captureException(new Error("my error"));
 
 ## Configuring the Scope
 
-There are two main ways to interact with the scope. You can access the current scope via `Sentry.getCurrentScope()` and use setters on the resulting scope, or you can use global methods like `Sentry.setTag()` directly, which will set on the respective scope under the hood.
+There are two main ways to interact with the scope. You can access the current scope via `Sentry.getCurrentScope()` and use setters on the resulting scope, or you can use global methods like `Sentry.setTag()` directly, which will set on the respective scope under the hood (which will be the current active isolation scope).
```

Comment:
```suggestion
There are two main ways to interact with the scope. You can access the current scope via `Sentry.getCurrentScope()` and use setters on the resulting scope, or you can use global methods like `Sentry.setTag()` directly, which will set on the respective scope under the hood (which will be the isolation scope).
```

I would use this wording, because `active` and `current` is overloaded a bit :)